clash-lib/src/Clash/Backend/VHDL.hs

```diff
@@ -781,6 +790,42 @@ funDec _ syn t@(Vector _ elTy) = Just
     eSz     = int (typeSize elTy)
     getElem = "islv" <> parens ("i * " <> eSz <+> "to (i+1) * " <> eSz <+> "- 1")
 
+funDec _ _ t@(MemBlob _ m) = Just
+  ( "function" <+> "toSLV" <+> parens ("value : " <+> qualTyName t) <+> "return std_logic_vector" <> semi <> line <>
+    "function" <+> "fromSLV" <+> parens ("slv" <+> colon <+> "in" <+> "std_logic_vector") <+> "return" <+> qualTyName t <> semi
+  , "function" <+> "toSLV" <+> parens ("value : " <+> qualTyName t) <+> "return std_logic_vector" <+> "is" <> line <>
+      indent 2
+        ( "alias ivalue    :" <+> qualTyName t <> "(1 to value'length) is value;" <> line <>
+          "variable result :" <+> "std_logic_vector" <> parens ("1 to value'length * " <> int m ) <> semi
+        ) <> line <>
+    "begin" <> line <>
+      indent 2
+        ("for i in ivalue'range loop" <> line <>
+            indent 2
+              (  "result" <> parens (parens ("(i - 1) * " <> int m) <+> "+ 1" <+>
+                                             "to i*" <> int m) <+>
+                          ":=" <+> ("ivalue" <> parens ("i")) <> semi
+              ) <> line <>
+         "end" <+> "loop" <> semi <> line <>
+         "return" <+> "result" <> semi
+        ) <> line <>
+    "end" <> semi <> line <>
+    "function" <+> "fromSLV" <+> parens ("slv" <+> colon <+> "in" <+> "std_logic_vector") <+> "return" <+> qualTyName t <+> "is" <> line <>
+      indent 2
+        ( "alias islv      :" <+> "std_logic_vector" <> "(0 to slv'length - 1) is slv;" <> line <>
+          "variable result :" <+> qualTyName t <> parens ("0 to slv'length / " <> int m <+> "- 1") <> semi
+        ) <> line <>
+    "begin" <> line <>
+      indent 2
+        ("for i in result'range loop" <> line <>
+            indent 2
+              ( "result" <> parens "i" <+> ":=" <+> "islv" <> parens ("i * " <> int m <+> "to (i+1) * " <> int m <+> "- 1") <> semi) <> line <>
+         "end" <+> "loop" <> semi <> line <>
+         "return" <+> "result" <> semi
+        ) <> line <>
+    "end" <> semi
+  )
```

Comment:
This can be removed if we just treat `MemBlobn n m` as an alias for `Vector n (BitVector m)`